orchestrator-salt/src/main/resources/salt/salt/postgresql/disaster_recovery/backup.sls

```diff
@@ -0,0 +1,9 @@
+include:
+  - postgresql.disaster_recovery
+
```

Comment:
Similarily check if it's external DB, otherwise, this state should not do anything.